tools/pom.xml

```diff
@@ -130,6 +130,24 @@
 				<version>1.2.0</version>
 			</dependency>
 
+			<dependency>
+				<groupId>javax.json.bind</groupId>
+				<artifactId>javax.json.bind-api</artifactId>
+				<version>1.0</version>
+			</dependency>
+
+			<dependency>
+				<groupId>org.eclipse</groupId>
+				<artifactId>yasson</artifactId>
+				<version>2.0.4</version>
+			</dependency>
+
+			<dependency>
+				<groupId>org.glassfish</groupId>
+				<artifactId>javax.json</artifactId>
+				<version>1.1.4</version>
+			</dependency>
+
```

Comment:
```suggestion
```

These really don't make sense to me… something's wrong

Reply:
i added those based on some sources i found -- e.g., "Eclipse Yasson is the reference implementation of the JSON-B 1.0 specification, and is the JSON-B implementation that Liberty provides by default when the jsonb-1.0 feature is enabled." https://www.ibm.com/docs/en/was-liberty/base?topic=manually-configuring-json-b-in-liberty

i'll test whether removing them works

Reply:
you're suggesting only change server.xml, no change needed to pom.xml? i'll try it...

Reply:
that's true, but you shouldn't need to load them manually.  In fact there's a danger because you might load a slightly different version than what the server already has